src/ray/gcs/gcs_server/gcs_actor_manager.cc

```diff
@@ -1132,5 +1142,27 @@ void GcsActorManager::AddDestroyedActorToCache(const std::shared_ptr<GcsActor> &
       actor->GetActorID(), (int64_t)actor->GetActorTableData().timestamp());
 }
 
+std::string GcsActorManager::DumpDebugMetrics() const {
+  std::ostringstream stream;
+  stream << "GcsActorManager metrics: "
+         << "\n{"
+         << "\nRegisterActor count: " << metrics_[MetricsType::REGISTER_ACTOR]
+         << "\nCreateActor count: " << metrics_[MetricsType::CREATE_ACTOR]
+         << "\nGetActorInfo count: " << metrics_[MetricsType::GET_ACTOR_INFO]
+         << "\nGetNamedActorInfo count: " << metrics_[MetricsType::GET_NAMED_ACTOR_INFO]
+         << "\nRegisterActorInfo count: " << metrics_[MetricsType::REGISTER_ACTOR_INFO]
+         << "\nUpdateActorInfo count: " << metrics_[MetricsType::UPDATE_ACTOR_INFO]
+         << "\nAddActorCheckpoint: " << metrics_[MetricsType::ADD_ACTOR_CHECKPOINT]
+         << "\nGetActorCheckpoint: " << metrics_[MetricsType::GET_ACTOR_CHECKPOINT]
+         << "\nGetActorCheckpointID: " << metrics_[MetricsType::GET_ACTOR_CHECKPOINT_ID]
+         << "\nRegistered actors count: " << registered_actors_.size()
+         << "\nDestroyed actors count: " << destroyed_actors_.size()
+         << "\nNamed actors count: " << named_actors_.size()
+         << "\nUnresolved actors count: " << unresolved_actors_.size()
+         << "\nPending actors count: " << pending_actors_.size()
+         << "\nCreated actors count: " << created_actors_.size() << "\n}";
```

Comment:
Having newline characters in a debug string is probably a bad idea. Please refer to other existing implementations of `DebugString`s. Please also make sure to use a consistent format.